tests/h/search/conftest.py

```diff
@@ -36,11 +37,21 @@ def index(es_client, pyramid_request):
     def _index(*annotations):
         """Index the given annotation(s) into Elasticsearch."""
         for annotation in annotations:
-            h.search.index.index(es_client, annotation, pyramid_request)
+            h.search.index.index_old(es_client, annotation, pyramid_request)
         es_client.conn.indices.refresh(index=es_client.index)
     return _index
 
 
+@pytest.fixture
+def index_new(pyramid_request):
+    def _index(*annotations):
+        """Index the given annotation(s) into Elasticsearch."""
+        for annotation in annotations:
+            h.search.index.index(annotation, pyramid_request)
+        elasticsearch_dsl.connections.get_connection().indices.refresh(index="hypothesis-test")
+    return _index
```

Comment:
Not sure what to do with the test fixtures. We want the same tests to run against both Es's ideally.